code/src/java/pcgen/io/IIFNode.java

```diff
@@ -31,15 +32,15 @@
  */
 class IIFNode
 {
-	private List<Object> falseChildren;
-	private List<Object> trueChildren;
-	private String expr;
+	private final List<Object> falseChildren;
+	private final List<Object> trueChildren;
+	private final String expr;
 
-	IIFNode(String _expr)
+	IIFNode(final String expr)
```

Comment:
One of the code style discussions we have had and do not do is put final on parameters.  While technically possible and correct, there is a feeling it is harder to read, and thus we do it when having it is absolutely necessary (e.g. used in an anonymous class), but not in the general case